src/pages/home/base/mobile-core/release-notes.md

```diff
@@ -8,6 +8,12 @@ keywords:
 
 # Release notes
 
+## December 13, 2023
+
+### Android Core 2.6.1
+
+* Fixed an issue where failure to create a webview for displaying an in-app message would cause the UI to be blocked.
```

Comment:
is there anything more specific we can mention with regards to the failure encountered when this issue occurred?

Reply:
No. We did not figure out the root cause of the failure. We only made changes to handle it gracefully.